plugins/cpu/cpu_osx

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+# -*- sh -*-
+
+# shellcheck disable=SC2046
+
+: << =cut
+
+=head1 NAME
+
+cpu - Plugin to measure cpu on osx.
+
+=head1 NOTES
+
+This plugin runs the top command once per interval, to discover the current value of CPU usage on OSX.
+The result is scaled to # of CPU's, so a 4 core machine will reach 400% utilization.
+Contributions are welcome to convert the plugin to use a long running counter such as /proc/stat in Linux.
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+NCPU=$(sysctl hw.ncpu | cut -d" " -f2)
+
+if [ "$1" = "autoconf" ]; then
+        echo yes
+        exit 0
+fi
+
+scaleto100=""
+
+if [ "$1" = "config" ]; then
+        # Linux had NCPU=$(egrep '^cpu[0-9]+ ' /proc/stat | wc -l)
+        if [ "$scaleto100" = "yes" ]; then
+                graphlimit=100
+        else
+                graphlimit=$(echo "$NCPU"*100 | bc)
```

Comment:
Please avoid using `bc`, since this would be an external dependencies (which you would need to check via `autoconf`). Instead you could just use `$((NCPU * 100))` (for integers) or `$(echo "$NCPU" | awk '{print($1 * 100)}')` for floating point numbers.